libkbfs/tlf_journal.go

```diff
@@ -417,9 +428,19 @@ func (j *tlfJournal) doBackgroundWorkLoop(bws TLFJournalBackgroundWorkStatus) {
 						"Background work error for %s: %v",
 						j.tlfID, err)
 
-					// TODO: Perhaps backoff and
-					// retry the background work
-					// again?
+					if j.retryTimer == nil {
```

Comment:
I think you can assume that `j.retryTimer` is always `nil` in this case, right? Maybe panic otherwise?